introduction.md

```diff
@@ -1,3 +1,14 @@
 Hi! My name is Melanie Hahn. I am an aspiring actor who really enjoys coding!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?